Client/examples/client_lib_basic_example.cpp

```diff
@@ -0,0 +1,120 @@
+    if (!conf_file_path.empty()) {
+        if (!confManager.load_from_file(conf_file_path)) {
+            std::cerr << "[ClientExample] Failed to load configuration file '" << conf_file_path << "'. Using default values instead." << std::endl;
+        } else {
+            std::cout << "[ClientExample] Configuration file loaded successfully from '" << conf_file_path << "'." << std::endl;
+        }
+    } else {
+        std::cout << "[ClientExample] No configuration file provided. Using default values." << std::endl;
+    }
+    confManager.log_configuration(std::cout);
+
+    // Initialize logging
+    int result = chronolog::chrono_monitor::initialize(confManager.LOG_CONF.LOGTYPE,
+                                                       confManager.LOG_CONF.LOGFILE,
+                                                       confManager.LOG_CONF.LOGLEVEL,
+                                                       confManager.LOG_CONF.LOGNAME,
+                                                       confManager.LOG_CONF.LOGFILESIZE,
+                                                       confManager.LOG_CONF.LOGFILENUM,
+                                                       confManager.LOG_CONF.FLUSHLEVEL);
+    if (result == 1) {
+        return EXIT_FAILURE;
+    }
+
+    // Build portal configs
+    chronolog::ClientPortalServiceConf portalConf;
+    portalConf.PROTO_CONF = confManager.PORTAL_CONF.PROTO_CONF;
+    portalConf.IP = confManager.PORTAL_CONF.IP;
+    portalConf.PORT = confManager.PORTAL_CONF.PORT;
+    portalConf.PROVIDER_ID = confManager.PORTAL_CONF.PROVIDER_ID;
+
+    // Build query configs
+    chronolog::ClientQueryServiceConf queryConf;
+    queryConf.PROTO_CONF = confManager.QUERY_CONF.PROTO_CONF;
+    queryConf.IP = confManager.QUERY_CONF.IP;
+    queryConf.PORT = confManager.QUERY_CONF.PORT;
+    queryConf.PROVIDER_ID = confManager.QUERY_CONF.PROVIDER_ID;
+
+    LOG_INFO("[ClientExample] Starting ChronoLog Client Example");
+
+    // Create a ChronoLog client
+    chronolog::Client client(portalConf, queryConf);
+
+    // Connect to ChronoVisor
+    int ret = client.Connect();
+    assert(ret == chronolog::CL_SUCCESS);
+
+    // Create a chronicle
+    std::string chronicle_name = "MyChronicle";
+    std::map<std::string, std::string> chronicle_attrs;
+    int flags = 0;
+    ret = client.CreateChronicle(chronicle_name, chronicle_attrs, flags);
+    assert(ret == chronolog::CL_SUCCESS);
+
+    // Acquire a story
+    std::string story_name = "MyStory";
+    std::map<std::string, std::string> story_attrs;
+    auto acquire_result = client.AcquireStory(chronicle_name, story_name, story_attrs, flags);
+    assert(acquire_result.first == chronolog::CL_SUCCESS);
+    auto story_handle = acquire_result.second;
+
+    // Log a few events to the story
+    story_handle->log_event("Event 1");
+    story_handle->log_event("Event 2");
+    story_handle->log_event("Event 3");
+
+    // Wait for events to be processed
+    sleep(120);
```

Comment:
Can you pls add a comment to explain this magic number saying this is to wait for the data to be persisted. and it's a temporary solution. future version may change or remove this value.